tree/dataframe/inc/ROOT/RDF/RResultMap.hxx

```diff
@@ -83,7 +83,7 @@ class RResultMap {
               std::vector<std::string> &&keys, ROOT::Detail::RDF::RLoopManager &lm,
               std::shared_ptr<ROOT::Internal::RDF::RActionBase> nominalAction,
               std::shared_ptr<ROOT::Internal::RDF::RActionBase> variedAction)
-      : fKeys{ROOT::Internal::RDF::Union({"nominal"}, keys)}, fLoopManager(&lm),
+      : fKeys{ROOT::Internal::RDF::Union({std::string("nominal")}, keys)}, fLoopManager(&lm),
```

Comment:
```suggestion
      : fKeys{ROOT::Internal::RDF::Union({"nominal"s}, keys)}, fLoopManager(&lm),
```
would be cleaner but then we'd need a `using namespace std::string_literals` somewhere. as you prefer.